npx-cli/README.md

```
@@ -114,6 +114,54 @@ Vibe Kanban supports customization through its configuration system:
+      "mcp_config_paths": ["/abs/tools.json", "/abs/cloud.json"],
+      "AUGGIE": { "command": { "base": "auggie", "params": ["--print"] } },
+      "auggie_model": "gpt-4o-mini",
+      "auggie_rules": ["/abs/rules/security.md", "/abs/rules/format.md"],
+      "auggie_augment_token_file": "/abs/token",
+      "variants": []
+    }
+  ]
+}
+```
+
+    }
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)